hudi-cli/pom.xml

```diff
@@ -254,12 +293,24 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.code.gson</groupId>
+          <artifactId>gson</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-hdfs</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Spring Boot, which is required by Spring Shell, have to use the newer version of gson. If not we will get an exception as below:

` java.lang.BootstrapMethodError: java.lang.NoSuchMethodError: com.google.gson.GsonBuilder.setLenient()Lcom/google/gson/GsonBuilder;`.

If we did not add it, a legacy version of `GSON` would be provided by Hadoop, which would lead to problems.